netgrasp/netgrasp.py

```diff
@@ -405,46 +412,43 @@ def wiretap(pc, child_conn):
                 run = False
         except Exception:
             ng.debugger.dump_exception("wiretap() while loop caught exception")
+
     ng.debugger.critical("No heartbeats from main process for >3 minutes, exiting.")
 
 
 def ip_on_network(ip):
+    import struct
+    import socket
+
     ng = netgrasp_instance
+    ng.debugger.debug("entering ip_on_network(%s)", (ip,))
 
     try:
-        import struct
-        import socket
-
-        ng.debugger.debug("entering address_on_network(%s)", (ip,))
-
         numeric_ip = struct.unpack("<L", socket.inet_aton(ip))[0]
         cidr = sum([bin(int(x)).count("1") for x in socket.inet_ntoa(ng.pcap["netmask"]).split(".")])
         netmask = struct.unpack("<L", ng.pcap["network"])[0] & ((2L<<int(cidr) - 1) - 1)
         return numeric_ip & netmask == netmask
-
     except:
-        ng.debugger.dump_exception("address_in_network() caught exception")
+        ng.debugger.dump_exception("ip_on_network() caught exception")
```

Comment:
ip stands for internet protocol. in this case, address (or ip_address) is more correct.